sms-broadcast/assets/index.html

```diff
@@ -51,7 +51,7 @@ <h2>Get started with your application</h2>
                     <ol class="steps">
                         <li>
                             In the Twilio Console page for your Notify Service,
```

Comment:
Should probably update this text too then. I imagine this was written this way in the first place because in the config page for a Notify service there is a direct link to create a new Messaging Service. I agree that this should point directly to Messaging Services though.